Service Tasks/deploy-a-dedicated-citrix-vpx-appliance.md

```diff
@@ -0,0 +1,53 @@
+{{{
+  "title": "Deploy a Dedicated Citrix VPX Appliance",
+  "date": "3-23-2015",
+  "author": "Chris Little",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Overview
+This knowledge base article will guide a customer through the process of deploying dedicated Citrix VPX load balancer(s) via [Service Task](http://www.centurylinkcloud.com/service-tasks) on the CenturyLink Cloud.
+
+### Deploy a Load Balancing VLAN
+
+* Deploy a [dedicated load balancing vlan](../Network/creating-and-deleting-vlans.md) in your account within the appropriate data center.  We recommend the VPX reside in a dedicated VLAN which allows for maximum Firewall security control and scalability of VIPs.
+* Costs for VLANs can be found in our [Pricing Catalog](http://www.centurylinkcloud.com/pricing) or your CenturyLink Cloud MSA.
+* Once this job completes we recommend you [apply a friendly name](../Network/add-a-user-friendly-name-to-vlans.md) to this VLAN.
+
+In the sample below in CA3 we have a network created and named NLB_10.100.97.0/24.
+
+![Dedicated Load Balancing VLAN Image](img src="../images/deploy-a-dedicated-citrix-VPX-appliance-01.png")
```

Comment:
You're mixing html syntax and markdown syntax. 
Markdown syntax for images looks like this: `![Dedicated Load Balancing VLAN Image](../images/deploy-a-dedicated-citrix-VPX-appliance-01.png)`

This is why the images are broken in the preview.
HTML equivalant looks like this: `<img src="../images/deploy-a-dedicated-citrix-VPX-appliance-01.png">`